docs/onboard-vendors-to-techpass.md

```diff
@@ -24,6 +24,7 @@ This section guides how vendors or contractors working on Government ICT Project
  <div class="warn">
  <ul>
  <li>SEED licence is applicable only for <b>non-GSIB device</b>.</li>
+ <li> Emails with <b>*_from@*.gov.sg</b> format are <b>not permitted</b>. Please provide a company email supplied by the vendor company</li>
```

Comment:
Can this note be added after step 2?
In step 2 we say enter your official email address. Instead of "company email", to be consistent, we can use the same terminology "official".

Reply:
it's not as obvious it seems... cos a _from@tech.gov.sg email is also an official email address..  so we still get many people attempting to sign up with it.. hence this additional note. i want to make it's absolutely clear right at the start.. it must be their vendor company email